bench/vectors/README.md

```diff
@@ -14,51 +15,52 @@ The generated executable will be in
 Its size is ~64MB . After stripping, the executable shrinks to just 11MB,
 but unfortunately it also stops running after stripping :-| .
 
-Compiling and running the V program:
-====================================
+## Compiling and running the V program:
+
 ```
 v crun vectors.v
 ```
-... produces and runs an executable `vectors` which is ~1.3MB in size.
 
+... produces and runs a `vectors` executable which is ~1.3MB in size.
 
 Compiling and running the V program, compiled with -prod:
+
 ```
 v -prod crun vectors.v
 ```
-... produces and runs an executable `vectors` which is ~176KB in size.
+
+... produces and runs a `vectors` executable which is ~176KB in size.
 After stripping, the executable shrinks to 157KB. It can still run after
 stripping.
 
-
 Note: using `crun` will make sure that the compilation will happen just
-once at the start, and then the executable will be just reused by the
-subsequent commands with identical options. It will also ensure that
-the compiled executable will not be removed, unlike `run` .
+once at the start, and then the executable will be reused by the subsequent
+commands with identical options. It will also ensure that the compiled
+executable will not be removed, unlike `run` .
```

Comment:
The previous wording was better imho. Please revert the whole file, and do not make changes that affect the meaning in a PR titled `fix typos` :-| .